python/lsst/ts/mtdome/mtdome_csc.py

```diff
@@ -40,26 +41,56 @@
 )
 
 _LOCAL_HOST = "127.0.0.1"
-_TIMEOUT = 20  # timeout in s to be used by this module
+_TIMEOUT = 20  # timeout [sec] to be used by this module
 _KEYS_TO_REMOVE = {
     "status",
     "operationalMode",  # Remove when next XML release is done
     "appliedConfiguration",  # Remove when next XML release is done
 }
-_KEYS_IN_RADIANS = {
+
+# The values of these keys need to be compensated for the dome azimuth offset
+# in the AMCS status. Note that these keys are shared with LWSCS so they can be
+# added to _KEYS_IN_RADIANS to avoid duplication but this also means that an
+# additional check for the AMCS lower level component needs to be done when
+# applying the offset correction. That is a trade off I can live with.
+_AMCS_KEYS_OFFSET = {
     "positionActual",
     "positionCommanded",
+}
+# The values of these keys need to be converted from radians to degrees when
+# the status is recevied and telemetry with these values is sent.
+_KEYS_IN_RADIANS = {
     "velocityActual",
     "velocityCommanded",
-}
+}.union(_AMCS_KEYS_OFFSET)
+
+# The offset of the dome rotation zero point with respect to azimuth 0º (true
+# north) is 32º west and the aperture lies at the opposite side of the dome.
+# Therefore the dome azimuth offset is 180º - 32º = 148º and this needs to be
+# subtracted when commanding the azimuth position, or added when sending the
+# azimuth telemetry.
+DOME_AZIMUTH_OFFSET = 148.0
```

Comment:
How stable is this likely to be? If it may change or require refinement, please consider making it a configuration parameter.

Reply:
This is 100% stable since it depends on the fixed position of the hardware locking pins. The only thing that may change is a refinement of the value. It could become 147.87 or 148.46 but otherwise this is fixed. I'll leave it hard coded.